### ISSUE TITLE
Compilation issue adding Mapsui to Android project

### ISSUE THREAD

**Issue by gpsaliola** [Mapsui/Mapsui#2080]
Mapsui Version: 4.0.0
Mapsui Platform: Android

Cannot add Mapsui.Forms to Android project:

Can not resolve reference: `Mapsui.UI.Forms`, referenced by `OkMap`. Please add a NuGet package or assembly reference for `Mapsui.UI.Forms`, or remove the reference to `OkMap`.			
Non è possibile trovare una versione di 'Svg.Skia' compatibile con 'Mapsui.Forms 4.0.0 vincolo: Svg.Skia (>= 0.6.0-rc1.1 && < 1.0.0)', 'Mapsui.Rendering.Skia 4.0.0 vincolo: Svg.Skia (>= 0.6.0-rc1.1 && < 1.0.0)'.				

My packages:

```
<?xml version="1.0" encoding="utf-8"?>
<packages>
  <package id="Amporis.Xamarin.Forms.ColorPicker" version="1.0.23" targetFramework="monoandroid12.1" />
  <package id="BruTile" version="5.0.4" targetFramework="monoandroid12.1" />
  <package id="SkiaSharp" version="2.88.3" targetFramework="monoandroid12.1" />
  <package id="SkiaSharp.NativeAssets.Android" version="2.88.3" targetFramework="monoandroid12.1" />
  <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Activity" version="1.2.4.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Annotation" version="1.2.0.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Annotation.Experimental" version="1.1.0.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.AppCompat" version="1.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.AppCompat.AppCompatResources" version="1.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Arch.Core.Common" version="2.1.0.9" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Arch.Core.Runtime" version="2.1.0.9" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.AsyncLayoutInflater" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Browser" version="1.3.0.6" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.CardView" version="1.0.0.9" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Collection" version="1.1.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Concurrent.Futures" version="1.1.0.3" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.ConstraintLayout" version="2.1.0" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.ConstraintLayout.Core" version="1.0.0" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.CoordinatorLayout" version="1.1.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Core" version="1.6.0.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.CursorAdapter" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.CustomView" version="1.1.0.7" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.DocumentFile" version="1.0.1.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.DrawerLayout" version="1.1.1.3" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.DynamicAnimation" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Fragment" version="1.3.6.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Interpolator" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Legacy.Support.Core.UI" version="1.0.0.9" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Legacy.Support.Core.Utils" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Legacy.Support.V4" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Lifecycle.Common" version="2.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Lifecycle.LiveData" version="2.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Lifecycle.LiveData.Core" version="2.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Lifecycle.Runtime" version="2.3.1.2" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Lifecycle.ViewModel" version="2.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Lifecycle.ViewModelSavedState" version="2.3.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Loader" version="1.1.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.LocalBroadcastManager" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Media" version="1.4.0.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Migration" version="1.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.MultiDex" version="2.0.1.5" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Navigation.Common" version="2.3.5.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Navigation.Runtime" version="2.3.5.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Navigation.UI" version="2.3.5.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Preference" version="1.1.1.9" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Print" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.RecyclerView" version="1.2.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.SavedState" version="1.1.0.2" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.SlidingPaneLayout" version="1.1.0.3" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.SwipeRefreshLayout" version="1.1.0.3" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Tracing.Tracing" version="1.0.0.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.Transition" version="1.4.1.1" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.VectorDrawable" version="1.1.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.VectorDrawable.Animated" version="1.1.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.VersionedParcelable" version="1.1.1.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.ViewPager" version="1.0.0.8" targetFramework="monoandroid12.1" />
  <package id="Xamarin.AndroidX.ViewPager2" version="1.0.0.10" targetFramework="monoandroid12.1" />
  <package id="Xamarin.Essentials" version="1.7.6" targetFramework="monoandroid12.1" />
  <package id="Xamarin.Forms" version="5.0.0.2578" targetFramework="monoandroid12.1" />
  <package id="Xamarin.Google.Android.Material" version="1.4.0.2" targetFramework="monoandroid12.1" />
  <package id="Xamarin.Google.Guava.ListenableFuture" version="1.0.0.2" targetFramework="monoandroid12.1" />
</packages>

```

**Comment by pauldendulk** [comment]
You mention Mapsui.UI.Forms (which is Xamarin Forms) and Android. Not sure what 'Android' means here. Is this a Xamarin.Android project? If so you should add Mapsui.Android not Mapsui.Forms. 

To further investigate this the best option is to create a new project and add Mapsui following the quickstart, so it is easy for us to reproduce your problem.

**Comment by gpsaliola** [comment]
I'm sorry. It's a Xamarin project for iOS and Android.
The issue was in Android project.
However I've removed all packages and changed:
`<TargetFrameworkVersion>v12.0</TargetFrameworkVersion>`
into
`<TargetFrameworkVersion>v11.0</TargetFrameworkVersion>`
Then I've added Xamarin.Forms and Xamarin.Essentials and compile was successfully done.
PS in packages.config I've changed targetFramework="monoandroid12.1" to targetFramework="monoandroid11.1".
